packages/react-integration/demo-app-ts/src/components/demos/AlertDemo/AlertCustomTimeoutDemo.tsx

```diff
@@ -26,7 +27,7 @@ export class AlertCustomTimeoutDemo extends React.Component<{}, AlertCustomTimeo
           Open Alert
         </Button>
         {isOpenAlert && (
-          <Alert id="alert-custom-timeout" title="custom timeout" timeout={16000}>
+          <Alert onTimeout={this.onTimeout} id="alert-custom-timeout" title="custom timeout" timeout={16000}>
```

Comment:
The alert should dismiss itself after the timeout, so setting isOpenAlert isn't necessary? The isOpenAlert appears to exist so the button can start the alert example.

Reply:
Yes, we can do that. But I am not sure cypress can catch `alert` messages.

Reply:
Done